bedrock/products/templatetags/misc.py

```diff
@@ -85,36 +84,24 @@ def vpn_subscribe_link(ctx, entrypoint, link_text, plan="12-month", class_name=N
     In Template
     -----------
 
-        {{ vpn_subscribe_link(entrypoint='www.mozilla.org-vpn-product-page', link_text='Get Mozilla VPN') }}
+        {{ vpn_subscribe_link(entrypoint='www.mozilla.org-vpn-product-page',
+                              link_text='Get Mozilla VPN',
+                              country_code=country_code,
+                              lang=LANG) }}
     """
 
-    # Set a default plan ID using page locale. This acts as a fallback should geo-location fail.
-    default_params = get_default_params(lang)
-    cc = get_cc(lang)
-    plan_default = default_params["default"][plan]["id"]
-    plan_attributes = {}
+    available_plans = get_available_plans(country_code, lang)
+    selected_plan = available_plans.get(plan)
+    plan_id = selected_plan.get("id")
 
-    # HTML data-attributes are used by client side JS to set the correct plan ID based on geo.
-    for country, attrs in settings.VPN_VARIABLE_PRICING.items():
-        if "alt" in attrs and cc in attrs["alt"]:
-            plan_id = attrs["alt"][cc][plan]["id"]
-        else:
-            plan_id = attrs["default"][plan]["id"]
-
-        plan_attributes.update({f"data-plan-{country}": plan_id})
-
-    if plan_attributes:
-        optional_attributes = optional_attributes or {}
-        optional_attributes.update(plan_attributes)
-
-    product_url = f"{settings.VPN_SUBSCRIPTION_URL}subscriptions/products/{settings.VPN_PRODUCT_ID}?plan={plan_default}"
+    product_url = f"{settings.VPN_SUBSCRIPTION_URL}subscriptions/products/{settings.VPN_PRODUCT_ID}?plan={plan_id}"
```

Comment:
So in theory `plan_id` could be `None` at this point (if config is absent). Is that OK?

Reply:
I guess theoreticallty that's possible yes, b ut pretty unlikely. I guess we could default to an empty string, which might be a better non-functional parameter value.

Reply:
After giving this some consideration, I think I'd rather have the page throw a 500 error if the `plan` is not a valid option, rather than default silently to something not so obvious value. This is easier to catch, and would almost certainly be picked up as a failure in a test run.